metadata.rb

```diff
@@ -1,6 +1,6 @@
 name              'logentries_agent'
 maintainer        'Logentries'
```

Comment:
```suggestion
maintainer        'Rapid7'
```
?

I'm actually not sure about this one, seeing as it's specifically a Logentries library.

Reply:
I'd say that while it is specifically a Logentries library, logentries doesn't exist anymore and the owner and maintainer is r7